controller/src/main/java/io/pravega/controller/store/client/StoreClientFactory.java

```diff
@@ -49,6 +51,12 @@ public static StoreClient createZKStoreClient(CuratorFramework client) {
     }
 
     private static CuratorFramework createZKClient(ZKClientConfig zkClientConfig) {
+        if (zkClientConfig.isSecureConnectionToZooKeeper()) {
+            System.setProperty("zookeeper.client.secure", "true");
+            System.setProperty("zookeeper.clientCnxnSocket", "org.apache.zookeeper.ClientCnxnSocketNetty");
+            System.setProperty("zookeeper.ssl.trustStore.location", zkClientConfig.getTrustStorePath());
```

Comment:
As these 4 config lines are replicated in several places, I would suggest to encapsulate them in a helper method (e.g., in `ZKUtils` or a new helper class) like: `setSecureZKClientProperties(String trustStorePath, String trustStorePassword)`.

Reply:
Fixed. Thanks.